PusherServer/ITriggerOptions.cs

```diff
@@ -9,5 +11,10 @@ public interface ITriggerOptions
         /// Gets or sets the Socket ID for a consuming Trigger
         /// </summary>
         string SocketId { get; set; }
+
+        /// <summary>
+        /// List of attributes that should be returned for each unique channel triggered to.
+        /// </summary>
+        List<string> Info { get; set; }
```

Comment:
This is also a BC. I was thinking about adding a new type that could be built via the builder pattern in order to prevent breakages when new options would be added, but I prefer to get some early feedack before changing to many things.